src/main/java/com/google/sps/servlets/DestinationDataServlet.java

```diff
@@ -0,0 +1,133 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.ArrayList;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.List;
+import java.util.Arrays;
+import java.util.stream.Collectors;
+import java.util.Set;
+import java.util.HashSet;
+
+/* Returns a destination created from user submitted information */
+@WebServlet("/destination-data")
+public class DestinationDataServlet extends HttpServlet {
+  
+  // Temporarily stores the destination created by the user
+  public List<com.google.sps.data.Destination> destinations = new ArrayList<>();
```

Comment:
I believe you can actually write `import com.google.sps.data.Destination;` at the top, and use `List<Destination> destinations` throughout the rest of this file (and same for Location, Riddle, etc.) -- this will make the code somewhat cleaner.

Reply:
Thanks I was wondering if there was a way for me to do that.

Reply:
^^ Agree with Lily's comment here!